src/Image.php

```diff
@@ -413,6 +413,35 @@ class Image extends ImageAutodoc implements \ArrayAccess
 {
 
     /**
+     * The logger instance.
+     *
+     * @var LoggerInterface
+     */
+    private static $logger;
+
+    /**
+     * Sets a logger.
+     *
+     * @param LoggerInterface $logger
+     *
+     * @return void
+     */
+    public static function setLogger(LoggerInterface $logger)
+    {
+        self::$logger = $logger;
+    }
+
+    /**
+     * Gets a logger.
+     *
+     * @return LoggerInterface $logger|null
+     */
+    public static function getLogger()
+    {
+        return self::$logger;
+    }
+
+    /**
```

Comment:
We could remove these unused lines, because they are defined in `Vips\Main` now.